src/network/netdev/bridge.c

```diff
@@ -230,6 +236,40 @@ int config_parse_bridge_port_priority(
                         prio);
 }
 
+int config_parse_bridge_fdb_max_learned(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+        Bridge *b = ASSERT_PTR(userdata);
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+        assert(data);
+
+        b->fdb_max_learned_set = false;
+
+        if (isempty(rvalue))
+                return 0;
```

Comment:
Please reset only when an empty string is passed. Otherwise, 
```
FDBMaxLearned=42
FDBMaxLearned=foo
```
will disable the first assignment. We usually ignore all invalid lines, but keep the previous valid assignments. So,
```c
if (isempty(rvalue)) {
        b->fdb_max_learned_set = false;
        return 0;
}
```